vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -308,31 +308,18 @@ def _make_page_404_layout(self):
         return html.Div(
             [
                 # Theme switch is added such that the 404 page has the same theme as the user-selected one.
-                html.Div(
-                    children=dbc.Switch(
-                        id="theme-selector",
-                        value=self.theme == "vizro_light",
-                        persistence=True,
-                        persistence_type="session",
-                    ),
-                    id="settings",
+                dbc.Switch(
+                    id="theme-selector",
+                    value=self.theme == "vizro_light",
+                    persistence=True,
+                    persistence_type="session",
                 ),
                 html.Img(src=f"data:image/svg+xml;base64,{error_404_svg}"),
-                html.Div(
```

Comment:
This nesting wasn't necessary anymore, as I changed the whole container to a flex box that is automatically center-aligned.

Reply:
💯 love this sort of change. If we compare our DOM now compared to 18 months ago the improvements are just HUGE.